docs/produce-events.md

```diff
@@ -153,8 +153,8 @@ See for instance this Java code that produces messages to a topic named “test
         }
     ```
 
-### Register with the Lighthouse Developer Hub
-The [Lighthouse Developer Hub](https://hub.lighthouse.va.gov/) is a software catalog that houses entities from across VA. Once you have your producer application up and running, it's important to register with the catalog to ensure that both current and future consumers can discover your event and access its details.
+### Register with CODE VA
+[CODE VA](https://hub.lighthouse.va.gov/) (formerly known as the Hub) is a software catalog that houses entities from across VA. Once you have your producer application up and running, it's important to register with the catalog to ensure that both current and future consumers can discover your event and access its details.
 
```

Comment:
Change Hub links to https://code.va.gov/.

Reply:
I will change this when code.va.gov is the production URL. Marking as resolved.